[2022-08-06 19:38:32,365] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 19:38:32,380] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 19:38:32,381] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:32,381] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:32,381] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:32,397] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-31 00:00:00+00:00
[2022-08-06 19:38:32,404] {standard_task_runner.py:52} INFO - Started process 2419 to run task
[2022-08-06 19:38:32,408] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '743', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4n4ri6lf', '--error-file', '/tmp/tmpeua0l_de']
[2022-08-06 19:38:32,408] {standard_task_runner.py:80} INFO - Job 743: Subtask second_task
[2022-08-06 19:38:32,478] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:32,603] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T00:00:00+00:00
[2022-08-06 19:38:32,604] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:32,605] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:32,622] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:32,624] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:32,625] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:32,665] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220331T000000, start_date=20220806T193832, end_date=20220806T193832
[2022-08-06 19:38:32,699] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:32,738] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:24,265] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 21:52:24,280] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 21:52:24,280] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:24,280] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:24,280] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:24,299] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-31 00:00:00+00:00
[2022-08-06 21:52:24,306] {standard_task_runner.py:52} INFO - Started process 2601 to run task
[2022-08-06 21:52:24,310] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '736', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpg5go_jvp', '--error-file', '/tmp/tmpk7uxa2s5']
[2022-08-06 21:52:24,310] {standard_task_runner.py:80} INFO - Job 736: Subtask second_task
[2022-08-06 21:52:24,406] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:24,522] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T00:00:00+00:00
[2022-08-06 21:52:24,523] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:24,524] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:24,542] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:24,544] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:24,544] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:24,605] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220331T000000, start_date=20220806T215224, end_date=20220806T215224
[2022-08-06 21:52:24,642] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:24,694] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:48,500] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 21:57:48,516] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 21:57:48,516] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:48,517] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:48,517] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:48,535] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-31 00:00:00+00:00
[2022-08-06 21:57:48,542] {standard_task_runner.py:52} INFO - Started process 2399 to run task
[2022-08-06 21:57:48,546] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '740', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpsj6c86vk', '--error-file', '/tmp/tmpr0se6ym3']
[2022-08-06 21:57:48,546] {standard_task_runner.py:80} INFO - Job 740: Subtask second_task
[2022-08-06 21:57:48,614] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:48,699] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T00:00:00+00:00
[2022-08-06 21:57:48,700] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:48,701] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:48,714] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:48,716] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:48,717] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:48,746] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220331T000000, start_date=20220806T215748, end_date=20220806T215748
[2022-08-06 21:57:48,797] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:48,832] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:50,617] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 22:12:50,631] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 22:12:50,632] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:50,632] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:50,632] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:50,650] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-31 00:00:00+00:00
[2022-08-06 22:12:50,658] {standard_task_runner.py:52} INFO - Started process 2411 to run task
[2022-08-06 22:12:50,661] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '739', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwgaeoud7', '--error-file', '/tmp/tmpb3irase6']
[2022-08-06 22:12:50,661] {standard_task_runner.py:80} INFO - Job 739: Subtask second_task
[2022-08-06 22:12:50,749] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:50,857] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T00:00:00+00:00
[2022-08-06 22:12:50,857] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:50,858] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:50,873] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:50,875] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:50,875] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:50,959] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220331T000000, start_date=20220806T221250, end_date=20220806T221250
[2022-08-06 22:12:50,992] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:51,040] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:09,005] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 22:24:09,020] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [queued]>
[2022-08-06 22:24:09,020] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:09,020] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:09,020] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:09,044] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-03-31 00:00:00+00:00
[2022-08-06 22:24:09,051] {standard_task_runner.py:52} INFO - Started process 2463 to run task
[2022-08-06 22:24:09,053] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-03-31T00:00:00+00:00', '--job-id', '742', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp5qw3anhw', '--error-file', '/tmp/tmp78j5byn9']
[2022-08-06 22:24:09,054] {standard_task_runner.py:80} INFO - Job 742: Subtask second_task
[2022-08-06 22:24:09,126] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-03-31T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:09,221] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-03-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-31T00:00:00+00:00
[2022-08-06 22:24:09,222] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:09,223] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:09,237] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:09,238] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:09,239] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:09,278] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220331T000000, start_date=20220806T222409, end_date=20220806T222409
[2022-08-06 22:24:09,305] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:09,347] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
